terraform/alb.tf

```diff
@@ -43,6 +43,9 @@ resource "aws_alb_target_group" "webservers_websockets" {
   protocol    = "HTTP"
   vpc_id      = aws_vpc.main.id
   target_type = "ip"
+  health_check {
```

Comment:
This is nice and declarative! Is this (health checks) something that is built into Rails?

Reply:
Nope, this is all in AWS.

I'll show you :)